tests/Fixer/Operator/LogicalOperatorsFixerTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Operator;
+
+use PhpCsFixer\Test\AbstractFixerTestCase;
+
+/**
+ * @author Haralan Dobrev <hkdobrev@gmail.com>
+ *
+ * @internal
+ */
+final class LogicalOperatorsFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @dataProvider provideFixCases
+     *
+     * @param mixed $expected
+     * @param mixed $input
+     * @param mixed $configuration
```

Comment:
@keradus I've updated it to `string`, `string`, `array<string, bool>` which is even more specific and I've seen the `array<>` annotation in the PHP CS Fixer codebase.